Maths/MagicSquareSeries.py

```diff
@@ -0,0 +1,2 @@
+def magicSquareSeries(n):
+    return n(n*n+1)/2
```

Comment:
this would cause _TypeError_ as it would interpret n as a function __n(__ _n*n+1_ __)__ _/2_

i think `n*(n*n+1)/2` is what you meant